package.json

```diff
@@ -43,13 +44,12 @@
         "history": "^4.5.1",
         "html-webpack-plugin": "^3.2.0",
         "inert": "^5.1.2",
-        "joi": "^14.3.1",
         "jwt-decode": "^2.1.0",
         "knex": "^0.17.3",
         "lodash": "^4.17.15",
         "material-components-web": "^0.16.0",
         "material-design-icons": "^3.0.1",
-        "material-ui": "^0.18.7",
+        "material-ui": "^0.20.2",
```

Comment:
This is risky. I know that there was a dropdown bug when upgrading, even though that was a year ago or so.

Reply:
Just tested this. If you click a dropdown it will flicker to the top left.

Reply:
Good memory, I'll revert